### PR TITLE
Bump @fluidframework/common-utils version

### PR DESCRIPTION
**Pull request by WayneFerrao**
[AB#646](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/646)

This bumps the @fluidframework/common-utils dependency version and updates garbageCollection.ts to use the util version of setLongTimeout.

### CODE REVIEW

File: examples/data-objects/focus-tracker/package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
   "dependencies": {
     "@fluid-experimental/data-objects": "^1.1.0",
     "@fluidframework/common-definitions": "^0.20.1",
-    "@fluidframework/common-utils": "^0.32.1",
+    "@fluidframework/common-utils": "^0.33.1000-0",
```
Comment (anthony-murphy) [inline examples/data-objects/focus-tracker/package.json:42]: @ssimic2 and @tylerbutler do we consider this a breaking change?

Comment (ssimic2) [reply examples/data-objects/focus-tracker/package.json:42]: Just bumping dependency may not be a breaking change. @tylerbutler ? 

@tylerbutler this is interesting from main-to-next integration. Would this commit go through without merge conflict ?  "next" should probably not be consuming "0.33.1000-0".

Comment (markfields) [reply examples/data-objects/focus-tracker/package.json:42]: If this were a definitions package I would think almost definitely yes should go in `next`.  But this is just a utils package.  Hopefully no API surface is returning a type defined in here - i.e. this thing should be all implementation details and be suitable for a minor release.  But maybe I'm over-simplifying it, or maybe we just want to play it safe :)

**Comment by WayneFerrao** [comment]
Closed this PR bc of conflicts and overlapping updates. Putting up new PR [here](https://github.com/microsoft/FluidFramework/pull/10778).

**Pull request closed without merging**